test/client/fixtures/core/page-unload-barrier-test.js

```diff
@@ -101,4 +102,52 @@ $(document).ready(function () {
                 start();
             });
     });
+
+    module('Regression');
+
+    if (browserUtils.isIE9 || browserUtils.isIE10) {
```

Comment:
this test should work in all browsers, not only in IE9 and IE10. Just delete this condition